parisienne/experimental-theme.json

```diff
@@ -0,0 +1,15 @@
+{
+	"color": {
+		"foreground": "#141414",
+		"background": "#F3EBE8",
+		"button-text": "white",
+		"button": "#817359",
```

Comment:
Why do we have a "button" and "button-text" variables, these seem too specific to me. Why . not just use "primary" for buttons and "light-text" or something for the button text (same for nav).

I think we should try to limit the number of variables as much as possible and compute the others.

Reply:
Those are good thoughts that I also agree with!

The first step was just extracting what Parisienne had and see how that looked. The next iteration for this PR should consolidate them into something general (for what we need to come up with a set of variables and its derivatives).

Reply:
I do agree that these seem overly specific for now. Feel free to just assign primary/background colors here. 

> I think we should try to limit the number of variables as much as possible and compute the others.

I agree that this makes sense for the defaults, but I do think it'll be important to make sure theme authors (and users) can override these computed values with custom ones on a theme + global styles level.